Header header
string name
JointDescription[] joints
LinkDescription[] links


